module1/homework/grayscaleImages/test.cpp

```diff
@@ -3,7 +3,7 @@
 #include <utility>  // for std::pair<>
 #include <vector>
 
-// TODO: include
+#include "compression.hpp"
```

Comment:
You could remove vector include here since it's in the header.

Reply:
+1